amico/core.py

```diff
@@ -59,6 +59,7 @@ def __init__( self, study_path, subject, output_path=None ) :
         self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
         self.set_config('doSaveCorrectedDWI', False)
+        self.set_config('doMergeB0', True) # Merge b0 volumes
```

Comment:
I think it is better to set it to `False` by default, this way it won't break the processing so far. Correspondingly, I will do the same in COMMIT for consistency.